app/models/ability.rb

```diff
@@ -66,6 +66,32 @@ def initialize(user)
         can :view_report_comments, Review
       end
 
+      if user.mdq_curator?
```

Comment:
I can't recall.  If the goal is to have the MDQ and ARC curators be entirely separated in their capabilities, I think that these also need to check that the item being acted on is from one of the appropriate DAACs.

Reply:
Have you written a ticket or talked to Alicia about this?

Reply:
This should be resolved, as user.mdq_curator? now checks daac as well.

Reply:
My concern is whether we need to validate (pseudocode) user.daac = <data>.daac

Reply:
Thomas did you end up writing a ticket for this?